src/utils/messages.js

```diff
@@ -73,6 +73,12 @@ export const UPDATE_REMINDER_MESSAGE = chalk.yellow(
     "Run `quest update` to download update.\n"
 );
 
+export const BREAKING_CHANGE_MESSAGE = chalk.yellow(
+  "Deprecated version of quest detected in local repo. New version introduces breaking changes.\n" +
+  "Run `quest find {QUEST-NAME}` to download breaking update.\n") +
+  chalk.bgBlackBright("Any of your existing work will be overwritten\n");
+
+
```

Comment:
Can we make this runtime dependent? So it will be a `breakingChangeMessage(questName)` function can returns the message with the `${questName}` embedded into the string